woocommerce/woocommerce
# [ci-jobs] Trigger test jobs for any dependency change

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/47267.

ci-jobs is not returning test jobs for parent project if a dependency is changed. Because of this some PRs got merged without relevant tests running.
This is a quick fix, a more elegant solution should be implemented in a follow-up.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Checkout this branch (and pull trunk into it if not already up to date) and change something in any of the woocommerce plugin dependencies. Run the ci-jobs utility and check the output. The e2e, api and performance tests for core should be included. 

`pnpm utils ci-jobs -r origin/trunk`
